src/components/LoanFinding/QuickFiltersSection.vue

```diff
@@ -67,6 +61,62 @@ export default {
 		},
 		handleQuickFiltersOverlay() {
 		},
+		async fetchFilterData(loanSearchState = {}) {
+			// TODO: Prevent this from running on every query (not needed for sorting and paging)
+			const { isoCodes } = await runFacetsQueries(this.apollo, loanSearchState, FLSS_ORIGIN_CATEGORY);
+			const categories = await fetchCategories(this.apollo);
+
+			// Merge all facet options with filtered options
+			const facets = {
+				regions: transformIsoCodes(isoCodes, this.allFacets?.countryFacets),
+			};
+
+			this.quickFiltersOptions.categories = [
+				...[{ title: 'All categories', key: 0 }],
+				...categories.lend?.loanChannels?.values
+			];
+
+			this.quickFiltersOptions.location = facets.regions;
+
+			this.quickFiltersOptions.sorting = [
+				{
+					title: 'Recommended',
+					key: 'personalized',
+				},
+				{
+					title: 'Almost funded',
+					key: 'amountLeft',
+				},
+				{
+					title: 'Amount high to low',
+					key: 'amountHighToLow'
+				},
+				{
+					title: 'Amount low to high',
+					key: 'amountLowToHigh'
+				},
+				{
+					title: 'Ending soon',
+					key: 'expiringSoon'
+				}
+			];
+			this.quickFiltersOptions.gender = [
+				{
+					title: 'All genders',
+					key: '',
+				},
+				{
+					title: 'Women',
+					key: 'female',
+				},
+				{
+					title: 'Men',
```

Comment:
Do we want the nonbinary option here as well? If so, the facets could be loaded from the API like in lend/filter.

Reply:
Same comment as regards to above - we will want to pull from an API after experiment phase

Reply:
In that case, we probably want to add nonbinary here for consistency with other filters. And I recommend adding a TODO here for returning to pull the data from the API.

Reply:
Yeah we currently support nonbinary in filtering on prod: https://www.kiva.org/lend/filter. One benefit of using the API is that it will return supported genders, so that should be the target longterm. But at the moment, it is safe to add nonbinary here as well manually.

Reply:
I'll add it manually for now and add a TODO note for integrating the API when we have this as a feature 👍